src/ex3/commands.js

```diff
@@ -0,0 +1,104 @@
+import { writeToFile, readFromFile } from "./fs.js";
+import {
+  checkItem,
+  capitalize,
+  convertToArray,
+  deletebyIndex,
+  deleteByName,
+} from "./utils.js";
+import { itemManager } from "../ex2/ItemManager.js";
+import { success, error, blue } from "./chalk.js";
+
+export async function addItem(item) {
+  let pokemonArray = [];
+
+  const inputArray = item.split(",");
+  inputArray.forEach(async (elm) => {
+    if (isNaN(elm)) {
+      await addToDo(elm);
+    } else {
+      pokemonArray.push(elm);
+    }
+  });
+
+  await addPokemon(pokemonArray);
+
+  async function addToDo(item) {
+    const capitalizedItem = capitalize(item);
+    await writeToFile(capitalizedItem, true, "a+");
+    console.log(success(`\n\nSuccessfully added: ${capitalizedItem}`));
+  }
+
+  async function addPokemon(pokemonArray) {
+    const allPromises = itemManager.createPromises(pokemonArray); // add promises to allPromises array
+    const pokemonData = await Promise.all(allPromises); // fetch all pokemons simulteniously
+
+    const copyPokemonArray = [...pokemonArray];
+
+    pokemonData.forEach((pokemon) => {
+      if (pokemon) {
```

Comment:
better to filter ahead of time and then you are running only on those that have a value